@codex fix this github actions build error, then update the branch and redeploy:
Run bun run typecheck
$ react-router typegen && tsc -b
app/articles/content/australian-ai-ecosystem/how-to-become-a-data-science.tsx(36,59): error TS2339: Property 'credentials' does not exist on type '{ name: string; role?: string | undefined; bio?: string | undefined; avatarUrl?: string | undefined; avatarAlt?: string | undefined; }'.
app/articles/content/featured/how-to-become-a-data-science.tsx(36,59): error TS2339: Property 'credentials' does not exist on type '{ name: string; role?: string | undefined; bio?: string | undefined; avatarUrl?: string | undefined; avatarAlt?: string | undefined; }'.
Error: Process completed with exit code 2.

--- a/app/articles/authors.ts
+++ b/app/articles/authors.ts
@@ -85,6 +85,7 @@ export function getAuthorProfile(key: AuthorRegistryKey): AuthorRegistryEntry {
 export function getDefaultArticleAuthorDetails(): {
     name: string;
     role?: string;
+    credentials?: string;
     bio?: string;
     avatarUrl?: string;
     avatarAlt?: string;
@@ -95,6 +96,7 @@ export function getDefaultArticleAuthorDetails(): {
     return {
         name: author?.name ?? fallbackName,
         role: author?.role,
+        credentials: author?.credentials,
         bio: author?.bio,
         avatarUrl: author?.avatarUrl ?? DEFAULT_AUTHOR_AVATAR_FALLBACK_URL,
         avatarAlt: author?.avatarAlt ?? author?.name ?? fallbackName,

